Software/app/basic_lorawan/main.c

```diff
@@ -47,6 +47,9 @@ int main(void)
   MX_LoRaWAN_Init();
   while (1)
   {
+#if defined(IWDG_TIMER_ON) && (IWDG_TIMER_ON == 1)
+    GNSE_BSP_IWDG_Refresh();

```

Comment:
This will break functionality in case of sleep and any transmission duty cycle change